packages/react-hooks/src/useQuestionnaireForm/useQuestionnaireForm.ts

```diff
@@ -213,6 +220,28 @@ export function useQuestionnaireForm(props: UseQuestionnaireFormProps): Readonly
     }
   }
 
+  function onChangeSignature(signature: Signature | undefined): void {
+    const currentResponse = state.current.questionnaireResponse;
+    if (!currentResponse) {
+      return;
+    }
+    if (signature) {
+      currentResponse.extension = currentResponse.extension ?? [];
+      currentResponse.extension = currentResponse.extension.filter(
+        (ext) => ext.url !== `${HTTP_HL7_ORG}/fhir/StructureDefinition/questionnaireresponse-signature`
```

Comment:
Consider moving to `utils.ts`

https://github.com/medplum/medplum/blob/main/packages/react-hooks/src/useQuestionnaireForm/utils.ts#L52-L57